From 6853cd7d7fedc665fcce95c01a939408ced9b708 Mon Sep 17 00:00:00 2001
From: Roman Stratiienko <r.stratiienko@gmail.com>
Date: Fri, 6 Oct 2023 23:15:17 +0300
Subject: [PATCH 2/2] HACK: Use mask in dri.sym

... otherwise LLD-17 will fail to link be because of missing symbols.

Signed-off-by: Roman Stratiienko <r.stratiienko@gmail.com>
---
 src/gallium/targets/dri/dri.sym | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/gallium/targets/dri/dri.sym b/src/gallium/targets/dri/dri.sym
index 6f502fa9f92..3080b0c979b 100644
--- a/src/gallium/targets/dri/dri.sym
+++ b/src/gallium/targets/dri/dri.sym
@@ -1,11 +1,11 @@
 {
 	global:
 		__driDriverGetExtensions*;
-		nouveau_drm_screen_create;
-		radeon_drm_winsys_create;
-		amdgpu_winsys_create;
-		fd_drm_screen_create_renderonly;
-		ac_init_shared_llvm_once;
+		nouveau_drm_screen_create*;
+		radeon_drm_winsys_create*;
+		amdgpu_winsys_create*;
+		fd_drm_screen_create_renderonly*;
+		ac_init_shared_llvm_once*;
 	local:
 		*;
 };
-- 
2.39.2

